diff_drive_controller:
  ros__parameters:
    left_wheel_names: ["left_wheel_joint"]
    right_wheel_names: ["right_wheel_joint"]

    wheel_separation: 0.3
    wheel_radius: 0.05

    publish_rate: 50
    cmd_vel_timeout: 0.5

    base_frame_id: base_link
    odom_frame_id: odom
    enable_odom_tf: true

    use_stamped_vel: false
    publish_cmd: true
